client/views/rules-editor.js

```diff
@@ -31,46 +31,78 @@ module.exports = View.extend({
   events: {
     'click [data-hook=rate-rule]' : 'addRule',
     'click [data-hook=assignment-rule]' : 'addRule',
-    'click [data-hook=save-rules]' : 'switchToViewMode',
     'click [data-hook=collapse]' : 'changeCollapseButtonText',
   },
-  initialize: function (args) {
+  initialize: function (attrs, options) {
     View.prototype.initialize.apply(this, arguments);
+    this.readOnly = attrs.readOnly ? attrs.readOnly : false;
     this.collection.parent.species.on('add remove', this.toggleAddRuleButton, this);
     this.collection.parent.parameters.on('add remove', this.toggleAddRuleButton, this);
     this.tooltips = Tooltips.rulesEditor
-    this.opened = args.opened
   },
   render: function () {
     View.prototype.render.apply(this, arguments);
-    this.renderRules();
-    this.toggleAddRuleButton()
-    if(this.opened) {
-      this.openRulesContainer();
+    if(this.readOnly) {
+      $(this.queryByHook('rules-edit-tab')).addClass("disabled");
+      $(".nav .disabled>a").on("click", function(e) {
+        e.preventDefault();
+        return false;
+      });
+      $(this.queryByHook('rules-view-tab')).tab('show');
+      $(this.queryByHook('edit-rules')).removeClass('active');
+      $(this.queryByHook('view-rules')).addClass('active');
+    }else {
+      this.renderEditRules();
+      this.toggleAddRuleButton();
     }
+    this.renderViewRules();
   },
   update: function () {
   },
   updateValid: function () {
   },
-  renderRules: function () {
+  renderEditRules: function () {
     if(this.rulesView) {
       this.rulesView.remove();
     }
     this.rulesView = this.renderCollection(
       this.collection,
       RuleView,
-      this.queryByHook('rule-list-container')
+      this.queryByHook('edit-rule-list-container')
     );
-    $(document).ready(function () {
+    $(function () {
+      $('[data-toggle="tooltip"]').tooltip();
+      $('[data-toggle="tooltip"]').click(function () {
+        $('[data-toggle="tooltip"]').tooltip("hide");
+      });
+    });
+  },
+  renderViewRules: function () {
+    if(this.viewRulesView) {
+      this.viewRulesView.remove();
+    }
+    this.containsMdlWithAnn = this.collection.filter(function (model) {return model.annotation}).length > 0;
+    if(!this.containsMdlWithAnn) {
+      $(this.queryByHook("rules-annotation-header")).css("display", "none");
+    }else{
+      $(this.queryByHook("rules-annotation-header")).css("display", "block");
+    }
+    let options = {viewOptions: {viewMode: true}};
+    this.viewRulesView = this.renderCollection(
+      this.collection,
+      RuleView,
+      this.queryByHook('view-rules-list-container'),
+      options
+    );
+    $(function () {
       $('[data-toggle="tooltip"]').tooltip();
       $('[data-toggle="tooltip"]').click(function () {
         $('[data-toggle="tooltip"]').tooltip("hide");
       });
     });
   },
   toggleAddRuleButton: function () {
-    this.renderRules();
+    // this.renderEditRules();
```

Comment:
Uncomment this

Reply:
Also replace `$(document).ready()` call on line 114

Reply:
Done and done